### ISSUE TITLE
Query fields list missing in the online Annotations/Queries docs

### ISSUE THREAD

**Issue by brucehoff** [Sage-Bionetworks/synapseDocs#577]
See: https://sagebionetworks.jira.com/browse/SWC-4924

```
Under the "Queries" section of the online Annotations and Queries doc page (https://docs.synapse.org/articles/annotation_and_query.html) there's a line that says "Along with annotations added by users, every entity has a number of fields useful for searching. For a complete list, see:", and then it looks like there should be a link to a list of query fields, but there's nothing there. See the attached screenshot.

The "SQL Query Examples" button takes me to a page with SQL examples, but it does not contain the query fields list.
```

**Comment by kelshmo** [comment]
@brucehoff Do we have a query fields list?

**Comment by brucehoff** [comment]
@john-hill might be able to provide this.

**Comment by kelshmo** [comment]
Discussion here: https://sagebionetworks.jira.com/browse/SWC-4924

**Comment by kdaily** [comment]
We could list:

- All entities (Projects, Files, Folders, Tables/Views, Docker containers): id, name, createdOn, createdBy, modifiedOn, modifiedBy, etag, type, parentId, benefactorId, projectId

- versionable entities (File, Table/View) get: currentVersion

- Files get: data filehandle ID

- File also have (in addition to the above, but NOT available from a view and are not searchable): md5, file size, content type

**Comment by kelshmo** [comment]
Closed by PR #654.
